mirrord-protocol/src/lib.rs

```diff
@@ -3,7 +3,10 @@
 #![feature(core_ffi_c)]
 
 pub mod codec;
+pub mod common;
 pub mod error;
+pub mod tcp;
 
 pub use codec::*;
+pub use common::*;
```

Comment:
Hmm, do we need this module right now just for some type aliases? I would just move those here.

Reply:
good idea